gstreamer/src/structure.rs

```diff
@@ -505,6 +505,38 @@ impl PartialEq for StructureRef {
 
 impl Eq for StructureRef {}
 
+impl<'a> ToGlibPtr<'a, *const ffi::GstStructure> for StructureRef {
```

Comment:
Hmm we don't implement that for any of the other `Ref` types either. Maybe just directly call `gst_structure_copy()` up there instead of implementing all these traits. We're not going to use them anywhere else probably.

Reply:
And it's leaner. Thanks